package.json

```diff
@@ -57,8 +57,13 @@
   },
   "dependencies": {
     "colr": "~1.2.1",
+    "prop-types": "^15.5.8",
     "rc-trigger": "1.x",
-    "rc-util": "2.x"
+    "rc-util": "^4.0.2"
+  },
+  "peerDependencies": {
+    "react": "^15.5.4",
```

Comment:
Should be a range https://docs.npmjs.com/misc/semver#ranges

Should support both 0.14.x and 15.x, right? @shepherdwind